src/main/kotlin/org/jetbrains/research/ml/dataset/anonymizer/util/FileUtil.kt

```diff
@@ -2,30 +2,54 @@ package org.jetbrains.research.ml.dataset.anonymizer.util
 
 import com.intellij.openapi.application.ApplicationManager
 import com.intellij.openapi.util.io.FileUtil
+import krangl.DataFrame
+import krangl.map
+import krangl.readCSV
+import krangl.writeCSV
+import org.jetbrains.research.ml.dataset.anonymizer.Anonymizer
+import org.jetbrains.research.ml.dataset.anonymizer.util.CodetrackerFileUtil.Language
 import java.io.File
 
 object FileUtil {
```

Comment:
Do we really need an object for that? Maybe we should have just functions since they don't share any properties?